### PR TITLE
V2 bump script

### PR DESCRIPTION
**Pull request by joannasese**
Took implementation from #507 and applied to v2.

### CODE REVIEW

**Comment by joannasese** [comment]
@bhelx I took what you did in #507 and tried to apply it to v2. Would like to discuss when you have a second. Not urgent.

File: .bumpversion.cfg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+[bumpversion:part:release]
+values = 
+	beta
+
+[bumpversion:part:build]
+
+[bumpversion:file:lib/recurly/version.rb]
+
+[bumpversion:file:docs/README.md]
```
Comment (bhelx) [inline .bumpversion.cfg]: Let's change this to `bumpversion:file:README.md`

File: .bumpversion.cfg [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	{major}.{minor}.{patch}.{release}.{build}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
+values = 
+	beta
+
+[bumpversion:part:build]
+
+[bumpversion:file:lib/recurly/version.rb]
```
Comment (bhelx) [inline .bumpversion.cfg]: For this to work, we need to refactor this file to be a normal string: https://github.com/recurly/recurly-client-ruby/blob/master/lib/recurly/version.rb#L3

File: .bumpversion.cfg [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+        (\.(?P<release>[a-z]+)\.(?P<build>\d+))?
+
+serialize = 
+	{major}.{minor}.{patch}.{release}.{build}
+	{major}.{minor}.{patch}
+
+[bumpversion:part:release]
```
Comment (bhelx) [inline .bumpversion.cfg]: we can move all the references to `build` and `release` as we should never have beta versions of this library.

File: .bumpversion.cfg [hunk 4]
```diff
@@ -0,0 +1,20 @@
+[bumpversion]
+current_version = 2.18.1
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+        (\.(?P<release>[a-z]+)\.(?P<build>\d+))?
+
+serialize = 
+	{major}.{minor}.{patch}.{release}.{build}
```
Comment (bhelx) [inline .bumpversion.cfg]: We can remove this line

File: .bumpversion.cfg [hunk 5]
```diff
@@ -0,0 +1,20 @@
+[bumpversion]
+current_version = 2.18.1
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+        (\.(?P<release>[a-z]+)\.(?P<build>\d+))?
```
Comment (bhelx) [inline .bumpversion.cfg]: We can remove this line

File: .bumpversion.cfg [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+parse = (?P<major>\d+)
+        \.(?P<minor>\d+)
+        \.(?P<patch>\d+)
+
+serialize = 
+	{major}.{minor}.{patch}
+
+[bumpversion:file:lib/recurly/version.rb]
+
+[bumpversion:file:README.md]
```
Comment (bhelx) [inline .bumpversion.cfg]: Nit: you might want to change your editor to make sure there is a newline at the end of every file. That's what the red circle is. It's not good practice to have an EOF on a non-empty line.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file
https://github.com/Microsoft/vscode/issues/12076#issuecomment-340037747

Comment (joannasese) [reply .bumpversion.cfg]: Done!

Screen Shot 2019-09-18 at 3 35 52 PM

**Pull request merged**